Sources/ObjCSupport/LOKOverlayLayout.swift

```diff
@@ -34,6 +35,7 @@ import Foundation
             backgroundLayouts: backgroundLayouts?.map { $0.unwrapped } ?? [],
             overlayLayouts: overlayLayouts?.map { $0.unwrapped } ?? [],
             alignment: self.alignment.alignment,
+            flexibility: flexibility?.flexibility ?? .flexible,
```

Comment:
Making this `flexibility?.flexibility ?? primaryLayouts.first?.flexibility ?? .flexible` should maintain backwards compatibility with the previous functionality where we took the first primary layout's flexibility if unspecified.